app/presenters/organisations/show_presenter.rb

```diff
@@ -148,6 +178,19 @@ def search_results_to_documents(search_results)
       }
     end
 
+    def contact_line(line)
+      return line + "<br/>" if line && line.length.positive?
+      ""
```

Comment:
Do we need to return `""` at all?

Reply:
Yes, because we're appending the return value onto an existing string. If the method returns `nil` it breaks. I could add more logic to get round this but I think this is the easiest approach.